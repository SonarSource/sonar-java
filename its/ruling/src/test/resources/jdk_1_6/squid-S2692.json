{
'jdk_1_6:java/beans/PropertyEditorManager.java':[
98,
],
}
