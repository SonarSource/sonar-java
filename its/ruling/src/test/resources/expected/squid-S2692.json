{
'project:com/sun/org/apache/bcel/internal/classfile/Signature.java':[
277,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionAvailableCall.java':[
131,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/FunctionCall.java':[
369,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Output.java':[
174,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/Parser.java':[
1015,
],
'project:com/sun/org/apache/xalan/internal/xsltc/runtime/StringValueHandler.java':[
110,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/DTDGrammar.java':[
550,
665,
],
'project:com/sun/org/apache/xml/internal/resolver/Resolver.java':[
377,
],
'project:com/sun/org/apache/xml/internal/resolver/helpers/Namespaces.java':[
50,
67,
],
'project:com/sun/org/apache/xml/internal/resolver/readers/SAXCatalogReader.java':[
322,
327,
],
'project:com/sun/org/apache/xml/internal/security/c14n/helper/C14nHelper.java':[
90,
],
'project:com/sun/org/apache/xml/internal/security/utils/CachedXPathFuncHereAPI.java':[
296,
346,
356,
],
'project:javax/sql/rowset/spi/SyncFactory.java':[
414,
],
}
